app/views/components/blockEditor.html

```diff
@@ -19,9 +19,9 @@
     <button class="btn btn-danger" ng-click="deleteBlock(block.id, true)" ng-disabled="!canDelete"><i class="fa fa-trash-o"></i> <translate>Delete</translate></button>
     <span class="btn btn-default" ui-tree-handle><i class="fa fa-bars"></i> <translate>Move</translate></span>
   </div>
-  <label>{{block.title}} <em class="required" ng-show="block.required">*</em></label>
+  <label ng-bind-html="block.title" tooltip="{{'Supports hyperlink html format.' | translate}}"><em class="required" ng-show="block.required">*</em></label>
```

Comment:
@OzzieOrca We have added a question mark next to the title in the edit panel which shows a popover with the message given by you. the example link stays in a single line.